libsrc/c128/kernal.s

```diff
@@ -4,6 +4,8 @@
 ; C128 kernal functions
 ;
 
+	.include "cbm_kernal.inc"
```

Comment:
There should be no /t chars whatsoever in the cc65 code base.

Reply:
Fixed